src/new-client/src/services/executions.ts

```diff
@@ -44,6 +46,8 @@ export enum ExecutionStatus {
   Ready = "Ready",
   Pending = "Pending",
   Done = "Done",
+  ClientTimeout = "ClientTimeout",
```

Comment:
I find a bit difficult to understand what `ClientTimeout` means. I would rather a more descriptive name like: `TakingTooLong`.